rasa_nlu/data_router.py

```diff
@@ -191,11 +206,10 @@ def get_status(self):
         # This will only count the trainings started from this process, if run in multi worker mode, there might
         # be other trainings run in different processes we don't know about.
         num_trainings = len(self.train_procs)
-        models = glob.glob(os.path.join(self.model_dir, '*'))
-        models = [model for model in models if os.path.isfile(os.path.join(model, "metadata.json"))]
+        agents = list(self.agent_store.keys())
         return {
             "trainings_under_this_process": num_trainings,
-            "available_models": models,
+            "available_models": agents,
```

Comment:
What do you mean exactly? Are you talking about the endpoint status or the underlying model?